spec/forms/developers_search_form_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+RSpec.describe DevelopersSearchForm do
+  describe 'Search Developers Function' do
+    before do
+      create(:programming_language, name: 'Python')
+      language_one = create(:language, code: 'en')
+      language_two = create(:language, code: 'vn')
+      create(:developer, email: 'tester@gmail.com', programming_languages: [create(:programming_language, name: 'PHP')], languages: [language_one, language_two])
+      create(:developer, email: 'leluongtruong7793@gmail.com', programming_languages: [create(:programming_language, name: 'Ruby')], languages: [language_two])
+    end
+
+    context 'Search with no search fields provided' do
```

Comment:
Normally should start with `when` or `with`.
Read through the `rspec` part of the following page.
https://github.com/jp-ryuji/setup_rails/pull/5